WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationDetailsViewController.m

```diff
@@ -61,13 +54,17 @@
 @interface NotificationDetailsViewController () <ReplyTextViewDelegate, SuggestionsTableViewDelegate>
 
 // Outlets
+@property (nonatomic,   weak) IBOutlet UIStackView          *stackView;
```

Comment:
FWIW, we have been using `strong` for IBOutlets throughout other areas of the app.